gradle.properties

```diff
@@ -33,5 +33,25 @@ mas_version=
 siteUrl = 'https://www.ca.com/us/developers/mas.html'
 gitUrl = 'https://github.com/CAAPIM/Android-MAS-SDK.git'
 issueUrl = 'https://github.com/CAAPIM/Android-MAS-SDK/issues'
+PUBLISH_GROUP_ID = 'com.ca.apim'
+version = '2.2'
+artifactId = '0.0'
```

Comment:
you set artifactId in each module's build config. what is this var in properties for?

Reply:
I just removed that..